.github/workflows/main.yml

```diff
@@ -1,11 +1,11 @@
 name: Release
 
 on:
-  push:
-  pull_request:
-  release:
-    types:
-      - published
+  # push:
+  # pull_request:
+  # release:
+  #   types:
+  #     - published
```

Comment:
```suggestion
  push:
  pull_request:
  release:
    types:
      - published
```

Reply:
Virer l'intégration continue pour supprimer les erreurs ? 🤷🏻
Ça les invisibilise, mais ça ne les enlève pas.

C'est quoi qui déconne ? Qu'on le corrige.

Reply:
Pas de soucis pour qu'on les corrige, juste que pour le moment, personne ne semble avoir le temps de s'en occuper (en tout cas ni moi ni @mrflos ) et ce script est alors bloquant...
Je propose donc de mettre de côté temporairement l'intégration continue pour permettre le debug de publication

Reply:
L'erreur est obtenue ici : https://github.com/YesWiki/yeswiki-extension-publication/runs/6580489875?check_suite_focus=true

```
Build container for action use: '/home/runner/work/_actions/oncletom/yeswiki-release-action/v1/Dockerfile'.
  /usr/bin/docker build -t 08450d:4c213d0db3ce4cab9357d2b8a5164f54 -f "/home/runner/work/_actions/oncletom/yeswiki-release-action/v1/Dockerfile" "/home/runner/work/_actions/oncletom/yeswiki-release-action/v1"
  Sending build context to Docker daemon  87.55kB
...
Step 3/8 : RUN apk add --update composer git jq zip npm     curl-dev libzip-dev     php7-sockets php7-mbstring php7-curl php7-mysqli php7-zip &&   docker-php-ext-install sockets &&   docker-php-ext-configure mbstring &&   docker-php-ext-configure curl --with-curl=/usr/include &&   docker-php-ext-configure mysqli &&   docker-php-ext-configure zip &&   apk del autoconf g++ libtool make
   ---> Using cache
   ---> b8d5935f65d9
  Step 4/8 : COPY composer.json /composer.json
   ---> Using cache
   ---> 5b00578420fa
  Step 5/8 : COPY composer.lock /composer.lock
   ---> Using cache
   ---> 0ffa1d480b1e
  Step 6/8 : RUN composer install --quiet
   ---> Running in 615a505a941c
  Your lock file does not contain a compatible set of packages. Please run composer update.
  The command '/bin/sh -c composer install --quiet' returned a non-zero code: 2
  
Error: Docker build failed with exit code 2
```

Reply:
Je ne sais pas, ... je n'ai pas regardé

Reply:
Tu as tenté de faire `composer update` ?

Reply:
oui je l'ai fait mais sur ma plateforme je n'ai pas `ext-sockets` donc je suis obligé de le faire tourner ainsi `composer update --ignore-platform-req=ext-sockets` et donc si `ext-sockets` n'est pas disponible sur le docker, il ne sera pas possible de faire `composer install --quiet`

Reply:
C'est installé mais on dirait que ça ne détecte pas l'extension (j'ai passé 1h en tournant le truc dans les sens, mais j'y vois pas clair). Ça m'a fait la même chose avec la libxml sur l'extension "test" du dépôt `yeswiki-release-action`.

Pourtant quand je fais `composer show --platform`, sockets et xml sont installés 🤷🏻

Sur ma machine (macOS), je ne rencontre pas le problème.
Donc y'a peut-être un souci sur la détection des extensions entre composer et alpine linux… je vais vérifier si y'a le même problème si la base d'image est une debian linux.

Reply:
Une question, est-ce que nous parlons du même composer ?
En effet, pour le `composer.json` de `yeswiki-extension-publication`, il sera exécuté par cette [ligne](https://github.com/thom4parisot/yeswiki-release-action/blob/8de72157fa8304226d2c22120f2166ca52a04523/package.sh#L69)
mais le bug apparait lors de l'exécution de cette [ligne](https://github.com/thom4parisot/yeswiki-release-action/blob/8de72157fa8304226d2c22120f2166ca52a04523/Dockerfile#L19) avant même le chargement de l'extension
Et dans le cas en question, `ext-sockets` n'est pas nécessaire vu que ça ne concerne que la partie `yeswiki-release-action`.
pour ce composer là, il faudra:
`ext-xml` `ext-json` `ext-simplexml`

Reply:
C'est le même problème : l'extension est installée, mais "pas détectée par composer" (c'est mon diagnostic, pas forcément le problème réel).

Reply:
ou peut-être qu'il nous faut lancer la commande
`RUN -e 'PHPRC=/right/path-to-folder-of-php-ini` composer install --quiet`